docs: improve documentation authenticity and naturalness (#6074)
web-infra-dev/rsbuild: website/docs/en/config/output/inline-styles.mdx

Whether to inline output style files (.css files) into HTML with `<style>` tags.

> When this option is enabled, style files are no longer generated in the dist directory and only exist inside the HTML file.

Reviewer: ```suggestion
When this option is enabled, style files are no longer written to the dist directory and only exist inside the HTML file.
```
